src/ImageSharp/ColorSpaces/Companding/SRgbCompanding.cs

```diff
@@ -18,49 +22,115 @@ namespace SixLabors.ImageSharp.ColorSpaces.Companding
     /// </remarks>
     public static class SRgbCompanding
     {
+        private const int Length = Scale + 2; // 256kb @ 16bit precision.
+        private const int Scale = (1 << 16) - 1;
+
+        private static readonly Lazy<float[]> LazyCompressTable = new Lazy<float[]>(() =>
+        {
+            var result = new float[Length];
+
+            for (int i = 0; i < result.Length; i++)
+            {
+                double d = (double)i / Scale;
+                if (d <= (0.04045 / 12.92))
+                {
+                    d *= 12.92;
+                }
+                else
+                {
+                    d = (1.055 * Math.Pow(d, 1.0 / 2.4)) - 0.055;
+                }
+
+                result[i] = (float)d;
+            }
+
+            return result;
+        });
+
+        private static readonly Lazy<float[]> LazyExpandTable = new Lazy<float[]>(() =>
+        {
+            var result = new float[Length];
+
+            for (int i = 0; i < result.Length; i++)
+            {
+                double d = (double)i / Scale;
+                if (d <= 0.04045)
+                {
+                    d /= 12.92;
+                }
+                else
+                {
+                    d = Math.Pow((d + 0.055) / 1.055, 2.4);
+                }
+
+                result[i] = (float)d;
+            }
+
+            return result;
+        });
```

Comment:
```suggestion
        }, LazyThreadSafetyMode.PublicationOnly);
```

Reply:
I've explicitly added the "true" second parameter.

Reply:
`true` is also redundant.  Thread-safe is the default mode: https://source.dot.net/#System.Private.CoreLib/Lazy.cs,236

Reply:
I remember checking that back in reference source when I did the other stuff but must have misread. I'll leave it for now as it's doing no harm. I can cleanup another time.